### ISSUE TITLE
BUG | AMR-82 | AMRPC cannot find iTunes

### ISSUE THREAD

**Issue by UnbreakCode** [zephraOSS/Apple-Music-RPC#26]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

AMRPC cannot find iTunes but it is installed
![image](https://user-images.githubusercontent.com/34512773/235622452-1424c977-12a7-4296-9ce0-883be5853eb8.png)

### Environment

```markdown
- OS: Windows 10 22H2
- Version:4.0.2-beta3
```

**Comment by N0chteil** [comment]
@UnbreakCode could you please upload the log file?

**Comment by UnbreakCode** [comment]
[main.log](https://github.com/zephraOSS/Apple-Music-RPC/files/11372460/main.log)

**Comment by N0chteil** [comment]
Thanks. Could you please try `where iTunes` and `gcm iTunes` in PowerShell and send the results?

**Comment by UnbreakCode** [comment]
I have found the problem:

iTunes is not working because of Apple Music Preview, that makes AM RPC not work. Maybe you could check if the AM Preview is installed before the checkIfAppIsInstalled

**Comment by N0chteil** [comment]
I have both installed, but I get a different result:
```
PS C:\Users\...> gcm iTunes

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     iTunes.exe                                         0.0.0.0    C:\Users\emila\AppData\Local\Microsoft...
```

**Comment by N0chteil** [comment]
> I have found the problem:
> 
> iTunes is not working because of Apple Music Preview, that makes AM RPC not work. Maybe you could check if the AM Preview is installed before the checkIfAppIsInstalled

Could you check if it works when Apple Music Preview isn't installed?

**Comment by N0chteil** [comment]
> > I have found the problem:
> > iTunes is not working because of Apple Music Preview, that makes AM RPC not work. Maybe you could check if the AM Preview is installed before the checkIfAppIsInstalled
> 
> Could you check if it works when Apple Music Preview isn't installed?

@UnbreakCode

**Comment by UnbreakCode** [comment]
> > > I have found the problem:
> > > iTunes is not working because of Apple Music Preview, that makes AM RPC not work. Maybe you could check if the AM Preview is installed before the checkIfAppIsInstalled
> > 
> > 
> > Could you check if it works when Apple Music Preview isn't installed?
> 
> @UnbreakCode

![image](https://user-images.githubusercontent.com/34512773/236151335-a78eebcb-942a-4b4a-b6fc-1aec55f7eb64.png)

**Comment by UnbreakCode** [comment]
iTunes is not in my list here
[gcm.txt](https://github.com/zephraOSS/Apple-Music-RPC/files/11395581/gcm.txt)

**Comment by N0chteil** [comment]
> iTunes is not in my list even after reinstall [gcm.txt](https://github.com/zephraOSS/Apple-Music-RPC/files/11395581/gcm.txt)

Maybe it's a problem on your end. I'll do some research

**Comment by UnbreakCode** [comment]
hm no I don't think so. Tried it on a diffrent PC with Windows 11 and have the same problem
![image](https://user-images.githubusercontent.com/34512773/236641023-d67d462c-4e9e-41a0-977c-13fc7f3a0216.png)

**Comment by N0chteil** [comment]
> hm no I don't think so. Tried it on a diffrent PC with Windows 11 and have the same problem ![image](https://user-images.githubusercontent.com/34512773/236641023-d67d462c-4e9e-41a0-977c-13fc7f3a0216.png)

Does `Get-Command` and/or `Get-Command iTunes` work?

**Comment by N0chteil** [comment]
> hm no I don't think so. Tried it on a diffrent PC with Windows 11 and have the same problem ![image](https://user-images.githubusercontent.com/34512773/236641023-d67d462c-4e9e-41a0-977c-13fc7f3a0216.png)

But `gcm` without `iTunes` works, right?

**Comment by UnbreakCode** [comment]
> But `gcm` without `iTunes` works, right?

yes

**Comment by N0chteil** [comment]
@UnbreakCode does `winget search Apple.iTunes` work?

**Comment by UnbreakCode** [comment]
> @UnbreakCode does `winget search Apple.iTunes` work?

yes

PS C:\WINDOWS\system32> winget search Apple.iTunes

Name   ID           Version   Quelle
iTunes Apple.iTunes 12.12.9.4 winget
PS C:\WINDOWS\system32>

**Comment by N0chteil** [comment]
- Fixed with 80a3eca
- Release fix: v4.2.1
